builder/scaleway/step_remove_volume.go

```diff
@@ -27,20 +27,22 @@ func (s *stepRemoveVolume) Cleanup(state multistep.StateBag) {
 	instanceAPI := instance.NewAPI(state.Get("client").(*scw.Client))
 	ui := state.Get("ui").(packersdk.Ui)
 	c := state.Get("config").(*Config)
-	volumeID := state.Get("root_volume_id").(string)
 
 	if !c.RemoveVolume {
 		return
 	}
 
-	ui.Say("Removing Volume ...")
-
-	err := instanceAPI.DeleteVolume(&instance.DeleteVolumeRequest{
-		VolumeID: volumeID,
-	})
-	if err != nil {
-		err := fmt.Errorf("error removing volume: %s", err)
-		state.Put("error", err)
-		ui.Error(fmt.Sprintf("Error removing volume: %s. (Ignored)", err))
+	ui.Say("Removing Volumes ...")
+
+	volumes := state.Get("volumes").([]*instance.VolumeServer)
+	for _, volume := range volumes {
+		err := instanceAPI.DeleteVolume(&instance.DeleteVolumeRequest{
+			VolumeID: volume.ID,
+		})
```

Comment:
Do you think parallelizing this would be useful and feasible ?

Reply:
DeleteVolume is an async action, it would need to waitForVolume to be sync.
Parallelizing is not useful here.